.storybook/pages/FeedbackOverview/FeedbackOverview.tsx

```diff
@@ -239,8 +239,8 @@ export const FeedbackOverview = ({ activeIndex = 0 }: Props) => {
                         <Table.HeaderCell>Final Product</Table.HeaderCell>
                       </Table.Row>
                       <Table.Row variant="header">
-                        <Table.Cell className="py-2"></Table.Cell>
-                        <Table.Cell className="py-2">
+                        <Table.Cell className="!py-2"></Table.Cell>
```

Comment:
Can you elaborate on why we need the `!important` modifiers now?

Reply:
It doesn't override the padding styles in TableCell.module.css:
```css
.table-cell {
  @mixin eds-theme-typography-body-text-sm;

  padding: var(--eds-size-2) var(--eds-size-1);
}
```
but now I'm thinking I might just add data attributes like `data-bootstrap-override="table-cell"` and override in LP instead of removing the `:where` pseudoclasses instead, so we wouldn't need the `!important` modifiers